peps/pep-0741.rst

```diff
@@ -20,16 +20,22 @@ used to add a built-in extension module; feature previously referred to
 as the "inittab".
 
 Add ``PyConfig_Get()`` and ``PyConfig_Set()`` functions to
-get and set the current runtime configuration at runtime.
+get and set the current runtime configuration.
 
 :pep:`587` "Python Initialization Configuration" unified all the ways to
 configure the Python **initialization**. This PEP unifies also the
 configuration of the Python **preinitialization** and the Python
-**initialization** in a single API.
+**initialization** in a single API. Moreover, this PEP only provides a
+single choice to embed Python, instead of having two "Python" and
+"Isolated" choices (:pep:`587`), to simplify the API further.
 
 This new API replaces the deprecated and incomplete legacy API which is
 scheduled for removal between Python 3.13 and Python 3.15.
 
+The :pep:`587` ``PyConfig`` API is soft deprecated: it should no longer
+be used to write new code, but it remains safe to continue using it in
+existing code.
```

Comment:
I don't think the soft-deprecation makes sense. Instead, I'd phrase this as something like:
```suggestion
The lower level :pep:`587` ``PyConfig`` API remains available for use cases with an
intentionally higher level of coupling to CPython implementation details (such as
emulating the full functionality of CPython's CLI, including its configuration
mechanisms).
```

Reply:
Ok, I applied your suggestion.